src/overmap.cpp

```diff
@@ -3020,25 +3020,37 @@ void overmap::signal_hordes( const tripoint &p, const int sig_power)
         if( !mg.horde ) {
             continue;
         }
+            // Horde has 33% chance to simply ignore signal.
+            if( one_in( 3 ) ) {
```

Comment:
It leads to one horde react one nuke time but other two on second nuke. Reason to make horde reaction less preditable for player.
"Do I lure them now or maybe not?"
"How fast do they come to me? Do they reach me ion one hour or I have more time?"
Also it is scramble horde reaction on continous shooting.